vpr/src/power/power.cpp

```diff
@@ -1261,11 +1261,11 @@ void power_routing_init(const t_det_routing_arch* routing_arch) {
         auto node = device_ctx.rr_nodes[rr_node_idx];
 
         for (t_edge_size edge_idx = 0; edge_idx < node.num_edges(); edge_idx++) {
-            if (node.edge_sink_node(edge_idx) != OPEN) {
-                if (rr_node_power[node.edge_sink_node(edge_idx)].driver_switch_type == OPEN) {
-                    rr_node_power[node.edge_sink_node(edge_idx)].driver_switch_type = node.edge_switch(edge_idx);
+            if (size_t(rr_graph.edge_sink_node(RRNodeId(rr_node_idx), edge_idx)) != size_t(OPEN)) {
```

Comment:
Using the new API, you do not need the ``OPEN``. This can be simplified as

```
if (rr_graph.edge_sink_node(RRNodeId(rr_node_idx), edge_idx)) {
```

Reply:
Sure I will do that on Monday.